src/layout/rectangles.js

```diff
@@ -56,6 +56,14 @@ export default function(layoutStrategy) {
                 return 'translate(' + offset.x + ', ' + offset.y + ')';
             });
 
+            g.each(function(d, i) {
```

Comment:
as discussed - I'd like to see whether the anchor can be written back to the bound data, rather than being associated with the DOM.